lib/src/components/NxDateInput/stateful/__tests__/NxStatefulDateInput.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Filename is missing the `.test.` part

Reply:
> Filename is missing the `.test.` part

Fixed: c6e1f4b31240cf0571402551b86aff7470a0e989